From a8e7c8cf78346cb788b7b9c84576ff989e031527 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Mon, 25 Aug 2014 07:51:23 -0400
Subject: [tools] perf/ui/browser: Remove misplaced __maybe_unused

Message-id: <1408953210-26343-196-git-send-email-jolsa@redhat.com>
Patchwork-id: 88975
O-Subject: [PATCH RHEL7.1 BZ1133083 195/322] perf ui browser: Remove misplaced __maybe_unused
Bugzilla: 1133083
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1133083
https://bugzilla.redhat.com/show_bug.cgi?id=1133083

upstream
========
commit 3184c47cb8b04b8bb5c1005168049519b066bcd1
Author: Arnaldo Carvalho de Melo <acme@redhat.com>
Date: Thu Dec 19 16:25:17 2013 -0300

description
===========
The 'browser' arg _is_ used, so ditch the misplaced attribute.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/ui/browser.c b/tools/perf/ui/browser.c
index cbaa7af..94223d4 100644
--- a/tools/perf/ui/browser.c
+++ b/tools/perf/ui/browser.c
@@ -268,7 +268,7 @@ int ui_browser__show(struct ui_browser *browser, const char *title,
  return err ? 0 : -1;
 }
 
-void ui_browser__hide(struct ui_browser *browser __maybe_unused)
+void ui_browser__hide(struct ui_browser *browser)
 {
  pthread_mutex_lock(&ui__lock);
  ui_helpline__pop();
-- 
1.7.1